pepdbagent/const.py

```diff
@@ -1,4 +1,4 @@
-DB_TABLE_NAME = "projects"
+DB_TABLE_NAME = "project"
```

Comment:
Why was this changed, our of curiosity?

Reply:
oops, it shouldn't be change! Thanks for this catch!